addon/templates/components/bootstrap-checkbox.hbs

```diff
@@ -3,7 +3,7 @@
     {{yield
       (hash
         label=(component control.label for=control.inputId srOnly=srOnly label=label)
-        input=(component 'bootstrap/-input'
+        input=(component 'bootstrap/-checkbox'
```

Comment:
On line 24 below this comment the following is defined:
```
{{bootstrap/-input
       tabindex=tabindex
       checked=isChecked
       onClick=(action "change")
       elementId=control.inputId
       type='checkbox'
       placeholder=placeholder
       readonly=readonly
       disabled=disabled
       key-press=key-press
       key-up=key-up
       key-down=key-down
       required=required
     }}
```

should this also be 'bootstrap/-checkbox' and not 'bootstrap/-input'?

Reply:
Yes it should, great catch Kevin.